src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/metrics_axes_options_helper.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ChartModes, ChartTypes, InterpolationModes } from '../utils/collections';
```

Comment:
```suggestion
import { ChartModes, ChartTypes, InterpolationModes, Positions  } from '../utils/collections';
```

Reply:
As we agreed offline, I moved it to [`utils.ts`](https://github.com/elastic/kibana/blob/e4d45ee7775239d08cf06b1d7fd80379e1af5c4b/src/legacy/core_plugins/kbn_vislib_vis_types/public/components/options/metrics_axes/utils.ts) since these util functions is used in the current folder.